runtime/runtime-domain-service/ballerina/APIClient.bal

```diff
@@ -3621,7 +3624,14 @@ public class APIClient {
                                 }
                                 updateAPI.operations = sortedOperations;
                                 _ = check self.updateAPI(apiId, updateAPI, validateAndRetrieveDefinitionResult.getContent(), organization, userName);
-                                return self.getAPIDefinitionByID(apiId, organization, APPLICATION_JSON_MEDIA_TYPE);
+                                http:Response|NotFoundError|PreconditionFailedError|InternalServerErrorError|commons:APKError response = self.getAPIDefinitionByID(apiId, organization, APPLICATION_JSON_MEDIA_TYPE);
+                                if response is http:Response {
```

Comment:
this doesn't need a location header

Reply:
Removed the location headers and updated `runtime-api.yaml`.